terraform/modules/infra-security-groups/main.tf

```diff
@@ -17,20 +16,22 @@ variable "additional_tags" {
 
 variable "aws_region" {
   type        = "string"
-  description = "AWS region"
-  default     = "eu-west-1"
+  description = "The AWS region to use."
```

Comment:
🤔 this variable is only needed for the remote state data source.  that got me thinking: is it maybe the project's job to wire up the remote state buckets?

Probably not worth doing in this PR but might be worth a discussion.